### DISCUSSION TITLE
Shared libraries need to be properly versioned on Linux

### DISCUSSION THREAD

**Discussion by crusader-mike** [aws/aws-sdk-cpp#3943] [General]
I.e. libFOO.so.\<major\>.\<minor\>, symlinks and etc. See some details here: http://stackoverflow.com/questions/7553184/how-to-do-versioning-of-a-shared-library-in-linux

**Comment by jmklix** [comment]
This is something we are actively working to improve. See more details here: https://github.com/aws/aws-sdk-cpp/issues/1888

**Comment by bkmgit** [comment]
This would be helpful for Fedora Linux. https://copr.fedorainfracloud.org/coprs/fed500/aws-sdk-cpp/builds/

**Comment by kai-ion** [comment]
Converting this to a Discussion. It's an improvements we plan to pursue over time rather than one actionable change with a near-term milestone, so a Discussion better fits collecting the specifics and design input.
